Helpers.ps1

```diff
@@ -800,3 +802,29 @@ function Get-HttpWebResponseContent
         }
     }
 }
+
+function Repair-LogFile
+{
+<#
+    .SYNOPSIS
+        Repais the log file encoding.
+
+    .DESCRIPTION
+        Repairs the log file by encoding it to UTF8 if it is currently UTF16.
+
+    .EXAMPLE
+        Repair-LogFile
+#>
+    $path = (Get-GitHubConfiguration -Name LogPath)
+
+    $logFileBytes = Get-Content -Encoding Byte -ReadCount 2 -Path $path
+
+    if ($logFileBytes -eq @(254, 255))
+    {
+        $logFileContent = [System.IO.File]::ReadLines($path)
```

Comment:
Hmmm...looking at this now, you may want to be [explicit with the expected encoding](https://docs.microsoft.com/en-us/dotnet/api/system.io.file.readlines?view=netcore-3.1#System_IO_File_ReadLines_System_String_System_Text_Encoding_) when reading the content.